docs/architecture.md

```diff
@@ -0,0 +1,292 @@
+# All About `armada-spark`
+
+## Running `armada-spark`
+
+`armada-spark`'s main entry point is a class called [`ArmadaSparkSubmit`](https://github.com/ClifHouck/armada-spark/blob/master/src/main/scala-spark-4.1/org/apache/spark/deploy/ArmadaSparkSubmit.scala).
```

Comment:
This probably shouldn't point to your repo.

Reply:
Addressed in 1dc03b4ad7656563ccce6a5fb9607fa2b8266382